cpu/stm32/periph/eth.c

```diff
@@ -104,40 +104,53 @@ netdev_t *_netdev;
 static uint8_t _link_state = LINK_STATE_DOWN;
 #endif
 
-/** Read or write a phy register, to write the register ETH_MACMIIAR_MW is to
- * be passed as the higher nibble of the value */
-static unsigned _rw_phy(unsigned addr, unsigned reg, unsigned value)
+/**
+ * @brief   Read or write a MII register
+ *
+ * @param[in]   addr    Which of the 32 possible PHY devices to access
+ * @param[in]   reg     MII register to access
+ * @param[in]   value   Value to write (ignored when @p write is `false`)
+ * @param[in]   write   Whether to write (`true`) or read (`false`) to/from the
+ *                      register
+ *
+ * @return  The value of the MII register accessed. (This should be equal to
+ *          @p value, if @p write was `true`.)
+ */
+static uint16_t _mii_reg_transfer(unsigned addr, unsigned reg, uint16_t value,
+                                  bool write)
 {
     unsigned tmp;
 
     while (ETH->MACMIIAR & ETH_MACMIIAR_MB) {}
     DEBUG("[stm32_eth] rw_phy %x (%x): %x\n", addr, reg, value);
 
-    tmp = (ETH->MACMIIAR & ETH_MACMIIAR_CR) | ETH_MACMIIAR_MB;
-    tmp |= (((addr & 0x1f) << 11) | ((reg & 0x1f) << 6));
-    tmp |= (value >> 16);
+    tmp = CLOCK_RANGE | ETH_MACMIIAR_MB | (((addr & 0x1f) << 11)
+          | ((reg & 0x1f) << 6));
```

Comment:
```suggestion
    tmp = CLOCK_RANGE | ETH_MACMIIAR_MB | (((addr & 0x1f) << 11)
        | ((reg & 0x1f) << 6));

```

Small whitespace tweaks to make it even more pleasant to the eye ;)